### PR TITLE
Dynamic import of ApproximateTerrainHeights data.

### PR DESCRIPTION
**Pull request by pmconne**
ApproximateTerrainHeights.initialize now does a dynamic import of ApproximateTerrainHeightsProps.ts instead of fetching ApproximateTerrainHeights.json from the server, where apps often didn't serve it from the expected location.
Really this serves as a simple test run to prove that we can also use dynamic imports for decoding of compressed meshes and textures.
NOTE: dynamic imports are only "dynamic" when using ESModules; applications (like certa) that choose CommonJS instead will get the "dynamically" imported code bundled directly into their app.

### CODE REVIEW

**Comment by wgoehrig** [comment]
@pmconne  This looks right to me

**Comment by pmconne** [comment]
> @pmconne This looks right to me

@wgoehrig certa doesn't like it. I can try to dig into the core-frontend tests webpack config but not sure what I'm looking for.
```
  1) ApproximateTerrainHeights
       test correct heights:
     ChunkLoadError: Loading chunk 0 failed.
(error: http://localhost:3000/0.bundled-tests.js)
      at Function.requireEnsure [as e] (webpack/bootstrap:105:1)
      at ApproximateTerrainHeights.initialize (D:\j2\s\imodeljs\core\frontend\src\ApproximateTerrainHeights.ts:42:37)
      at Context.call (D:\j2\s\imodeljs\core\frontend\src\test\tile\map\ApproximateTerrainHeights.test.ts:21:26)
      at callFn (@/lib/runnable.js:366:21)
      at Test.run (@/lib/runnable.js:354:5)
      at Runner.runTest (@/lib/runner.js:677:10)
      at fn (@/lib/runner.js:801:12)
      at next (@/lib/runner.js:594:14)
      at fn (@/lib/runner.js:604:7)
      at next (@/lib/runner.js:486:14)
```

File: core/frontend/src/test/utils/webpack.config.js [hunk 1]
```diff
@@ -55,6 +55,10 @@ function createConfig(shouldInstrument) {
             env[key] = JSON.stringify(process.env[key]);
             return env;
           }, {}),
+      }),
+      // Disable code-splitting for tests - see dynamic import in ApproximateTerrainHeights.ts.
```
Comment (pmconne) [inline core/frontend/src/test/utils/webpack.config.js:59]: @wgoehrig this resolves the following error  when running certa - look reasonable?
```
  1) ApproximateTerrainHeights
       test correct heights:
     ChunkLoadError: Loading chunk 0 failed.
(error: http://localhost:3000/0.bundled-tests.js)
      at Function.requireEnsure [as e] (webpack/bootstrap:105:1)
      at ApproximateTerrainHeights.initialize (D:\j2\s\imodeljs\core\frontend\src\ApproximateTerrainHeights.ts:42:37)
      at Context.call (D:\j2\s\imodeljs\core\frontend\src\test\tile\map\ApproximateTerrainHeights.test.ts:21:26)
      at callFn (@/lib/runnable.js:366:21)
      at Test.run (@/lib/runnable.js:354:5)
      at Runner.runTest (@/lib/runner.js:677:10)
      at fn (@/lib/runner.js:801:12)
      at next (@/lib/runner.js:594:14)
      at fn (@/lib/runner.js:604:7)
      at next (@/lib/runner.js:486:14)
```

Comment (pmconne) [reply core/frontend/src/test/utils/webpack.config.js]: Reverted.

**Review by markschlosseratbentley — APPROVED** [review]

**Review by bbastings — APPROVED** [review]

**Pull request merged**